modules/administration-guide/partials/assembly_configuring-server-logging.adoc

```diff
@@ -0,0 +1,21 @@
+:parent-context-of-configuring-server-logging: {context}
+
+[id="configuring-sever-logging_{context}"]
+= Configuring server logging
+
+:context: configuring-server-logging
+
+It is possible to fine-tune the log levels of individual loggers available in the {prod-short} server.
+
+The log level of the whole {prod-short} server is configured globally using the xref:installation-guide:configuring-the-che-installation.adoc#checluster-custom-resources-fields-reference_configuring-the-che-installation[`cheLogLevel` configuration property] of the Operator.
+To set the global log level in installations not managed by the Operator, specify the `CHE_LOG_LEVEL` environment variable in the `che`
```

Comment:
`CHE_LOG_LEVEL`->`CHE_LOGS_LEVEL`

Reply:
No, you are right. That CHE_LOG_LEVEL transformed into CHE_LOGS_LEVEL in `entrypoint.sh`